### PR TITLE
added handle submit to login

### PR DESCRIPTION
**Pull request by Arjun-1080**

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -74,6 +74,9 @@ function App() {
       </Routes>
       <FooterBar/>
     </Router>
+    // <>
```
Comment (AaryanNarayani) [inline client/src/App.tsx:77]: Remove this unceccesary comments

File: client/src/pages/(admin)/AdminPage.tsx [hunk 2]
```diff
@@ -1,13 +1,31 @@
...
+      name : 'Hotel Samudra',
+      address : ''
+    }
+  
+  ];
+   
+    
   
+
+
```
Comment (AaryanNarayani) [inline client/src/pages/(admin)/AdminPage.tsx:17]: Unneccesary white spaces

**Review by Nischit-Ekbote — COMMENTED** [review]
bhari be

File: client/src/App.tsx (hunk 1 above)
Comment (Arjun-1080) [reply client/src/App.tsx:77]: Thik hai bhau, chuk zhali

**Pull request merged**